scripts/install-nix-from-closure.sh

```diff
@@ -202,18 +202,7 @@ added=
 p=$HOME/.nix-profile/etc/profile.d/nix.sh
 if [ -z "$NIX_INSTALLER_NO_MODIFY_PROFILE" ]; then
     # Make the shell source nix.sh during login.
-    for i in .bash_profile .bash_login .profile; do
-        fn="$HOME/$i"
-        if [ -w "$fn" ]; then
-            if ! grep -q "$p" "$fn"; then
-                echo "modifying $fn..." >&2
-                echo -e "\nif [ -e $p ]; then . $p; fi # added by Nix installer" >> "$fn"
-            fi
-            added=1
-            break
-        fi
-    done
-    for i in .zshenv .zshrc; do
+    for i in .bash_profile .bashrc .bash_login .profile .zshenv .zshrc; do
```

Comment:
Note that we only add the value to one of these (see break below).

Reply:
ah, thanks!